### PR TITLE
Clean up repository

### PR DESCRIPTION
**Pull request by idaknow**
- Removed showing the average of 3 eye predictions to showing every prediction
- Changed SweetAlert from bug fixed repository to update one downloaded through NPM
- Fixed function declarations for SweetAlert (they'd changed)
- Removed unnecessary files/ comments from our code changes
- Changed documentation to be more like BrownHCI but inc. our changes

### CODE REVIEW

**Comment by idaknow** [comment]
[Test our current demonstration HTML](https://trello.com/c/nFSdIa81/8-test-our-current-demonstration-html)

File: README.md [hunk 1]
```diff
@@ -46,13 +50,6 @@ The following browsers support WebGazer.js:
 
 Your browser needs to support the getUserMedia API as seen [here](http://caniuse.com/#feat=stream).
 
-## Windows Support
```
Comment (Libby713) [inline README.md]: I wonder if we shouldn't remove this?

File: dependencies/sweetalert/LICENSE [hunk 2]
```diff
@@ -1,22 +0,0 @@
-The MIT License (MIT)
```
Comment (Libby713) [inline dependencies/sweetalert/LICENSE]: I think we have to keep this copyright note in

File: js/main.js [hunk 3]
```diff
@@ -1,11 +1,9 @@
 window.onload = function() {
+
     //start the webgazer tracker
     webgazer.setRegression('ridge') /* currently must set regression and tracker */
         .setTracker('clmtrackr')
         .setGazeListener(function(data, clock) {
-            //log the data
```
Comment (Libby713) [inline js/main.js]: did we write this? I didn't and if nobody else did I don't think we should remove it

File: dependencies/sweetalert/LICENSE (hunk 2 above)
Comment (idaknow) [reply dependencies/sweetalert/LICENSE]: Resolved - it's in the node_modules repo

File: README.md (hunk 1 above)
Comment (idaknow) [reply README.md]: Resolved - I added this

File: js/main.js (hunk 3 above)
Comment (idaknow) [reply js/main.js]: Resolved - added back in

**Pull request merged**